package.json

```diff
@@ -19,6 +19,7 @@
   },
   "scripts": {
     "docs": "(cd showcase && npm run build && cp bundle.* .. && cd .. && git checkout gh-pages && git commit -am 'Upgrade docs' && git push && git checkout master)",
```

Comment:
i guess my other question is how with the gitbook interface with the doc script? we need to make sure that this pipeline is easy to run